### ISSUE TITLE
Remove dependency to pytables

### ISSUE THREAD

**Issue by fmaussion** [OGGM/oggm#1754]
This package has been a pain since years, causing troubles everywhere (cc @facusapienza21 @JordiBolibar)

This will need a bit of detective work because its needed for pandas to open hdf files. 

We also need to find another format to store tabular data instead - maybe parquet? To investigate.

**Comment by JordiBolibar** [comment]
This would be super high on our list. We've been having many issues with
our Julia-Python interface, mainly due to OpenSSL and related dependencies
changing in a recent Julia release. On the Python side this is impacting us
mainly through pytables from OGGM. Facu has been following this recently,
so he'll be able to comment more accurately.

El dv., 13 de des. 2024, 21:10, Fabien Maussion ***@***.***>
va escriure:

**Comment by fmaussion** [comment]
I discussed briefly with Facu. 

I also wondered if you could also simply remove all dependency to OGGM (or python) by simply using its glacier directories without calling python at all. The few bits of logic that decide which data to download and open based on a RGI ID are trivial, and could be recoded in Julia in relatively little time. As long at julia knows how to download tar files and open them...

**Comment by JordiBolibar** [comment]
Yes, this would be the ideal solution. Today I briefly discussed this with
the new postdoc at IGE working on ODINN. We should have a meeting about
this next year when I'm back :)

El dv., 13 de des. 2024, 21:43, Fabien Maussion ***@***.***>
va escriure:
